contracts/reserve-auction/src/msg.rs

```diff
@@ -1,17 +1,42 @@
 use cosmwasm_schema::{cw_serde, QueryResponses};
-use cosmwasm_std::{Coin, Timestamp, Uint128};
+use cosmwasm_std::Coin;
+use sg_marketplace_common::query::QueryOptions;
 
-use crate::state::{Auction, Config};
+use crate::state::{Auction, Config, HaltManager};
 
 #[cw_serde]
 pub struct InstantiateMsg {
-    pub marketplace: String,
-    pub min_reserve_price: Coin,
-    pub min_duration: u64,
+    // The address of the fair burn contract
```

Comment:
This is documentation that we want rustdoc to pick up right? Then it should use `///` instead of `//`.